src/scripting/daedalus/DaedalusVMForGameWorld.hpp

```diff
@@ -93,7 +99,7 @@ namespace REGoth
     public:
       REGOTH_DECLARE_RTTI(DaedalusVMForGameWorld);
 
-    public:                              // FIXME: RTTI protected
+    public:                                // FIXME: RTTI protected
```

Comment:
We should take care of the remaining RTTI-protected FIXME in another PR. It's not straightforward for bs::IReflectable. We also befriend bs::SceneObject inside REGOTH_DECLARE_RTTI, which is only correct for components.

Reply:
We probably need separate `DECLARE` and `DEFINE` macros for `IReflectable` and  `Component` (and probably also a seperate one for pure abstract `IReflectables`). Doing it for `IReflectable` is pretty straight-forward (see `RaycastShadowSampler` in the shadow-sampling branch) and could also be encapsulated in a macro.